lms/djangoapps/student_account/views.py

```diff
@@ -67,16 +62,6 @@ def login_and_registration_form(request, initial_mode="login"):
     # Retrieve the form descriptions from the user API
     form_descriptions = _get_form_descriptions(request)
 
-    # If this is a microsite, revert to the old login/registration pages.
-    # We need to do this for now to support existing themes.
-    # Microsites can use the new logistration page by setting
-    # 'ENABLE_COMBINED_LOGIN_REGISTRATION' in their microsites configuration file.
-    if microsite.is_request_in_microsite() and not microsite.get_value('ENABLE_COMBINED_LOGIN_REGISTRATION', False):
-        if initial_mode == "login":
-            return old_login_view(request)
-        elif initial_mode == "register":
-            return old_register_view(request)
```

Comment:
Yes, I would leave this check in for now and turn on the `ENABLE_COMBINED_LOGIN_REGISTRATION` flag for microsites. Please add the flag [here](https://github.com/edx/edx-microsite/blob/master/mit-prof-ed/CONFIG_NOTES.txt) and create a DevOps support ticket to have them update the stage/prod MICROSITE_CONFIGURATIONs in the secure configuration repo.